dll/cpl/sysdm/lang/ro-RO.rc

```diff
@@ -34,236 +34,236 @@ END
-    DEFPUSHBUTTON "Con&firmă", IDOK, 141, 258, 50, 15
-    PUSHBUTTON "A&nulează", IDCANCEL, 195, 258, 50, 15
+    AUTOCHECKBOX "Se s&uprascrie orice fişier existent", IDC_STRRECOVERWRITE, 17, 228, 155, 10
+    DEFPUSHBUTTON "OK", IDOK, 141, 258, 50, 15
+    PUSHBUTTON "Revocare", IDCANCEL, 195, 258, 50, 15
 END
 
 IDD_VIRTMEM DIALOGEX 6, 6, 223, 248
 STYLE DS_SHELLFONT | DS_MODALFRAME | DS_CONTEXTHELP | WS_POPUP | WS_VISIBLE | WS_CAPTION | WS_SYSMENU
 CAPTION "Memorie virtuală"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    LTEXT "Unitate [etichetă volum]", IDC_STATIC, 12, 5, 96, 9
-    LTEXT "Limite de paginare (Mo)", IDC_STATIC, 135, 5, 96, 9
+    LTEXT "&Unitate [Etichetă volum]", IDC_STATIC, 12, 5, 96, 9
+    LTEXT "Dimensiune fişier de paginare (MO)", IDC_STATIC, 135, 5, 96, 9
     LISTBOX IDC_PAGEFILELIST, 10, 16, 204, 52, LBS_NOTIFY | LBS_USETABSTOPS
-    GROUPBOX "Dimensiune fișier de paginare (din selecție)", IDC_DRIVEGROUP, 10, 70, 204, 104
+    GROUPBOX "Dimensiune fişier de paginare pentru unitatea selectată", IDC_DRIVEGROUP, 10, 70, 204, 104
     LTEXT "Unitate:", IDC_STATIC, 20, 80, 50, 8
     LTEXT "%s", IDC_DRIVE, 88, 80, 100, 9
-    LTEXT "Spațiu disponibil:", IDC_STATIC, 20, 92, 56, 9
+    LTEXT "Spaţiu disponibil:", IDC_STATIC, 20, 92, 56, 9
     LTEXT "%s", IDC_SPACEAVAIL, 88, 92, 104, 9
-    LTEXT "Di&mensiunea inițială (Mo):", -1, 32, 118, 87, 9
-    LTEXT "Dimensiunea ma&ximă (Mo):", -1, 32, 131, 87, 9
-    AUTORADIOBUTTON "Limite parti&cularizate:", IDC_CUSTOM, 20, 105, 154, 9, WS_GROUP
-    AUTORADIOBUTTON "&Limite gestionate de sistem", IDC_SYSMANSIZE, 20, 145, 188, 9
-    AUTORADIOBUTTON "Fă&ră paginare", IDC_NOPAGEFILE, 20, 158, 78,9
+    LTEXT "D&imensiune iniţială (MO):", -1, 32, 118, 87, 9
+    LTEXT "Dimensiune ma&ximă (MO):", -1, 32, 131, 87, 9
+    AUTORADIOBUTTON "&Dimensiune particularizată:", IDC_CUSTOM, 20, 105, 154, 9, WS_GROUP
+    AUTORADIOBUTTON "Di&mensiune gestionată de sistem", IDC_SYSMANSIZE, 20, 145, 188, 9
+    AUTORADIOBUTTON "&Fără fişier de paginare", IDC_NOPAGEFILE, 20, 158, 78,9
     EDITTEXT IDC_INITIALSIZE, 120, 114, 44, 13, NOT WS_BORDER, WS_EX_CLIENTEDGE
     EDITTEXT IDC_MAXSIZE, 120, 129, 44, 13, NOT WS_BORDER, WS_EX_CLIENTEDGE
-    PUSHBUTTON "&Aplică", IDC_SET, 158, 154, 50, 15
-    GROUPBOX "Paginare", IDC_TOTALGROUP, 10, 177, 204, 46
-    LTEXT "Minimă permisă:", IDC_STATIC, 18, 188, 58, 9
+    PUSHBUTTON "&Stabilire", IDC_SET, 158, 154, 50, 15
+    GROUPBOX "Dimensiune totală fişier de paginare pentru toate unităţile", IDC_TOTALGROUP, 10, 177, 204, 46
+    LTEXT "Minim permis:", IDC_STATIC, 18, 188, 58, 9
     LTEXT "%s", IDC_MINIMUM, 88, 188, 100, 9
-    LTEXT "Recomandată:", IDC_STATIC, 18, 199, 52, 9
+    LTEXT "Recomandat:", IDC_STATIC, 18, 199, 52, 9
     LTEXT "%s", IDC_RECOMMENDED, 88, 199, 100, 9
-    LTEXT "Actuală:", IDC_STATIC, 18, 210, 66, 9
+    LTEXT "Alocat în prezent:", IDC_STATIC, 18, 210, 66, 9
```

Comment:
Please attach a pic that this does not overflow the 66 width.

Reply:
![2024-05-05 17_58_12-Resource Hacker - sysdm cpl mui](https://github.com/reactos/reactos/assets/113553683/775c7b1f-db68-4b9d-a2c9-51af4cf1fc76)

Reply:
Resolved, despite the fact that this is the MS pic. But if MS doesn't overflow at 65, then we won't overflow wih 66 neither.